### ISSUE TITLE
【Android】No implementation found for method scanBarcode on channel flutter_barcode_scanner

### ISSUE THREAD

**Issue by NewComerBH** [AmolGangadhare/flutter_barcode_scanner#10]
```
E/flutter (22956): [ERROR:flutter/lib/ui/ui_dart_state.cc(148)] Unhandled Exception: MissingPluginException(No implementation found for method scanBarcode on channel flutter_barcode_scanner)
E/flutter (22956): #0      MethodChannel.invokeMethod (package:flutter/src/services/platform_channel.dart:300:7)
E/flutter (22956): <asynchronous suspension>
E/flutter (22956): # 1      FlutterBarcodeScanner.scanBarcode (package:flutter_barcode_scanner/flutter_barcode_scanner.dart:27:43)
E/flutter (22956): <asynchronous suspension>
E/flutter (22956): # 2      _MyAppState.initPlatformState (package:flutter_barcode_scanner_example/main.dart:28:39)
E/flutter (22956): <asynchronous suspension>
E/flutter (22956): # 3      _MyAppState.build.<anonymous closure>.<anonymous closure> (package:flutter_barcode_scanner_example/main.dart:59:23)
E/flutter (22956): # 4      _InkResponseState._handleTap (package:flutter/src/material/ink_well.dart:511:14)
E/flutter (22956): # 5      _InkResponseState.build.<anonymous closure> (package:flutter/src/material/ink_well.dart:566:30)
```

```
flutter doctor -v 
[✓] Flutter (Channel stable, v1.5.4-hotfix.2, on Mac OS X 10.14.4 18E226, locale
    zh-Hans-CN)
    • Flutter version 1.5.4-hotfix.2 at
• Framework revision 7a4c33425d (11 days ago), 2019-04-29 11:05:24 -0700
    • Engine revision 52c7a1e849
    • Dart version 2.3.0 (build 2.3.0-dev.0.5 a1668566e5)

[✓] Android Studio (version 3.4)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 35.0.1
    • Dart plugin version 183.6270
    • Java version OpenJDK Runtime Environment (build
      1.8.0_152-release-1343-b01)
```

**Comment by AmolGangadhare** [comment]
Have you cloned it or using it from the pub? Can you provide some steps you followed?
Note: try upgrading dependencies when you open example if you cloned it.

**Comment by fikurniawan** [comment]
it's also happening to me, using it from the pub with the last dependency. First run I can open the camera and its scan, when I close the app and run again, the camera is not open anymore.

**Comment by AmolGangadhare** [comment]
@fikurniawan I was not able to reproduce this. Can you provide any logs? Also the devices you tried it on?

**Comment by fikurniawan** [comment]
So I run with the last pub, and try to copy paste your example, run it and scan with no problem. Then I close the app, open it then try to scan again but it's not open the camera. Using Samsung S8.

[flutter doctor.txt](https://github.com/AmolGangadhare/flutter_barcode_scanner/files/3459501/flutter.doctor.txt)
[log error.txt](https://github.com/AmolGangadhare/flutter_barcode_scanner/files/3459502/log.error.txt)

**Comment by AmolGangadhare** [comment]
@fikurniawan have you tried `flutter clean`? Do you mind to perform some preliminary steps to solve this? Just follow these steps.

1. If you are using Android Studio just try `Invalidate cache and Restart once`
2. open cmd/ terminal goto project and do `flutter clean`
3. Go to pubspec and add the latest version of the plugin which is `0.1.5` and click pug packages get.
4. Run project.

After performing these steps please acknowledge that if the issue fixed or not?

Thanks

**Comment by AmolGangadhare** [comment]
please refer to #7

**Comment by AmolGangadhare** [comment]
Closing this as it is resolved in #7
